tests/testthat/test-plot-grid.R

```diff
@@ -0,0 +1,52 @@
+test_that("plots grid is rendered correctly", {
+  skip_if_not_installed("vdiffr")
+  skip_if_not_installed("ggplot2")
+  skip_if_not_installed("tlf")
+  skip_if(getRversion() < "4.1")
+
+  library(ggplot2)
+  library(tlf)
```

Comment:
I think `library(ggplot2)` is not required - tlf loads ggplot automatically

Reply:
Are you sure about that? Because _ggplot2_ doesn't show up in the search environments if I only load _tlf_:

``` r
library(tlf)
rlang::search_envs()
#>  [[1]] $ <env: global>
#>  [[2]] $ <env: package:tlf>
#>  [[3]] $ <env: package:stats>
#>  [[4]] $ <env: package:graphics>
#>  [[5]] $ <env: package:grDevices>
#>  [[6]] $ <env: package:utils>
#>  [[7]] $ <env: package:datasets>
#>  [[8]] $ <env: package:methods>
#>  [[9]] $ <env: Autoloads>
#> [[10]] $ <env: tools:callr>
#> [[11]] $ <env: package:base>
```

<sup>Created on 2022-04-07 by the [reprex package](https://reprex.tidyverse.org) (v2.0.1)</sup>

Otherwise:

``` r
library(tlf)
library(ggplot2)
rlang::search_envs()
#>  [[1]] $ <env: global>
#>  [[2]] $ <env: package:ggplot2>
#>  [[3]] $ <env: package:tlf>
#>  [[4]] $ <env: package:stats>
#>  [[5]] $ <env: package:graphics>
#>  [[6]] $ <env: package:grDevices>
#>  [[7]] $ <env: package:utils>
#>  [[8]] $ <env: package:datasets>
#>  [[9]] $ <env: package:methods>
#> [[10]] $ <env: Autoloads>
#> [[11]] $ <env: tools:callr>
#> [[12]] $ <env: package:base>
```

<sup>Created on 2022-04-07 by the [reprex package](https://reprex.tidyverse.org) (v2.0.1)</sup>

Reply:
that's true. however e.g. 
```
library(tlf)
tlf::plotHistogram(x = rnorm(100))
```
works (and after that ggplot2 is still not shown as loaded). don't ask me why :)

Reply:
It certainly doesn't load _ggplot2_, so we should leave the `library(ggplot2)` call in.

``` r
library(tlf)
ggplot(mtcars, aes(wt, mpg)) + geom_point()
#> Error in ggplot(mtcars, aes(wt, mpg)): could not find function "ggplot"
```

<sup>Created on 2022-04-07 by the [reprex package](https://reprex.tidyverse.org) (v2.0.1)</sup>